.pre-commit-config.yaml

```
@@ -0,0 +1,36 @@
+---
+repos:
+
+  # Code formatter for both python files and jupyter notebooks
+  - repo: https://github.com/psf/black
+    rev: 22.10.0
+    hooks:
+      - id: black-jupyter
+      - id: black
+        language_version: python3.10
+
```

```suggestion
repos:
  # Formats import order
  - repo: https://github.com/pycqa/isort
    rev: 5.12.0
    hooks:
      - id: isort
        name: isort (python)

  # Code formatter for both python files and jupyter notebooks
  - repo: https://github.com/psf/black
    rev: 22.10.0
    hooks:
      - id: black-jupyter
      - id: black
        language_version: python3.10

```

I would also recommend adding `isort` into your pre-commit as it sorts your imports in the PEP8 standard!